cartridges/int_extend_sfra/cartridge/client/default/js/product/base.js

```diff
@@ -665,8 +666,10 @@ module.exports = {
                     form.extendPlanId = extendPlan.planId;
                     form.extendPrice = extendPlan.price;
                     form.extendTerm = extendPlan.term;
+                    isPlanSelected = true;
                     $(this).trigger('updateAddToCartFormData', form);
-                } else if (EXT_PDP_UPSELL_SWITCH) {
+                }
+            } if (EXT_PDP_UPSELL_SWITCH && !isPlanSelected) {
```

Comment:
else if?

Reply:
I don't see any difference after this change, What is changed?

Reply:
'Else if' was not used because 'else if' executes only if 'if' condition was false and 'else if' one is true. But we need to execute statements from both conditions.